src/silx/io/url.py

```diff
@@ -37,6 +37,37 @@
 _logger = logging.getLogger(__name__)
 
 
+def slice_to_string(s):
+    """Convert a Python slice into a string"""
+    if s == Ellipsis:
+        return "..."
+    elif isinstance(s, slice):
+        result = ""
+        if s.start is None:
+            result += ":"
+        else:
+            result += f"{s.start}:"
+        if s.stop is not None:
+            result += f"{s.stop}"
+        if s.step is None:
+            result += ""
+        else:
+            result += f":{s.step}"
+        return result
+    elif isinstance(s, int):
+        return str(s)
+    else:
+        raise TypeError("Unexpected slicing type. Found %s" % type(s))
+
+
+def slice_sequence_to_string(data_slice):
```

Comment:
Same, base private:
```suggestion
def _slice_sequence_to_string(data_slice):
```

Reply:
I have to reuse it at another place. That's why it's public

Reply:
All right then.